src/SFML/Window/CMakeLists.txt

```diff
@@ -346,3 +346,14 @@ elseif(SFML_OS_IOS)
 elseif(SFML_OS_ANDROID)
     target_link_libraries(sfml-window PRIVATE android)
 endif()
+
+# on some platforms (e.g. Raspberry Pi 3 armhf), GCC requires linking libatomic to use <atomic> features
+# that aren't supported by native CPU instructions (64-bit atomic operations on 32-bit architecture)
+if(SFML_COMPILER_GCC)
+    include(CheckCXXSourceCompiles)
+    check_cxx_source_compiles("#include <atomic>
+        int main(){std::atomic_ullong x(1); return x.fetch_add(1);}" ATOMIC_TEST)
```

Comment:
Probably makes no difference, but since we're using `std::atomic<…>` below, wouldn't it be better to use that in this test, too? Over the years I've seen some rather interesting halfway-complete headers on some MinGW versions for example.

Reply:
This is what was already added to `CMakeLists.txt` for sfml-graphics, we didn't want to include `<cstdint>`. Since `std::atomic_ullong` is just a type alias to `std::atomic<unsigned long long>` I really don't see why it could have been left out when they added atomic support. There are some atomic types that are optional according to the standard, but this isn't one of them.